CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing
+This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [code of conduct](CODE_OF_CONDUCT.md).
+
+## Reporting Issues and Asking Questions
+Before opening an issue, please search the [issue tracker](https://github.com/shakacode/bootstrap-loader/issues) to make sure your issue hasn't already been reported.
+
+## Development
+Fork, then clone the repo:
+
+```
+git clone https://github.com/your-username/bootstrap-loader.git
+```
+
+To start development simply run:
+
+```
+npm start
+```
+
+It will run linters, clear directory with previous build, create new build and run watchers to re-build on every change. Sadly, but we can't use `npm link` feature in development process here because this command symlinks package folder, what makes impossible to resolve `bootstrap` packages in project's `npm_modules` folder. Instead of this just install `bootstrap-loader` locally:
+
+```
+cd my-test-project
+npm install --save-dev ../path/to/local/bootstrap-lodaer
+```
+
+It is a pity, but you have to re-install `bootstrap-lodaer` on every change.
```

Comment:
> typo for bootstrap-loader

:+1: 

> why not use npm-link instead so you don't have to reload on every change to bootstrap-loader? If that doesn't work, we should say so.

It's in the previous paragraph (I guess I need to break it in 2):

> It will run linters, clear directory with previous build, create new build and run watchers to re-build on every change.

> Sadly, but we can't use `npm link` feature in development process here because this command symlinks package folder, what makes impossible to resolve `bootstrap` packages in project's `npm_modules` folder. Instead of this just install `bootstrap-loader` locally: